src/overlays/actors/ovl_En_Hy/z_en_hy.c

```diff
@@ -61,242 +69,380 @@ static ColliderCylinderInit D_80A71EC0 = {
+    /* 14 */ { &gObjOsAnim_41F8, 1.0f, 0.0f, -1.0f, 0x00, 0.0f },
+    /* 15 */ { &gObjOsAnim_300C, 1.0f, 0.0f, -1.0f, 0x00, 0.0f },
+    /* 16 */ { &gObjOsAnim_31B0, 1.0f, 0.0f, -1.0f, 0x00, 0.0f },
+    /* 17 */ { &gObjOsAnim_31B0, 1.0f, 0.0f, -1.0f, 0x00, -8.0f },
+    /* 18 */ { &gObjOsAnim_2D0C, 1.0f, 0.0f, -1.0f, 0x00, 0.0f },
+    /* 19 */ { &gObjOsAnim_2DC0, 1.0f, 0.0f, -1.0f, 0x00, 0.0f },
+    /* 20 */ { &gObjOsAnim_4408, 1.0f, 0.0f, -1.0f, 0x00, 0.0f },
+    /* 21 */ { &gObjOsAnim_1F18, 1.0f, 0.0f, -1.0f, 0x00, 0.0f },
+    /* 22 */ { &gObjOsAnim_4F28, 1.0f, 0.0f, -1.0f, 0x00, 0.0f },
+    /* 23 */ { &gObjOsAnim_33B4, 1.0f, 0.0f, -1.0f, 0x00, -8.0f },
+    /* 24 */ { &gObjOsAnim_12E8, 1.0f, 0.0f, -1.0f, 0x00, -8.0f },
+    /* 25 */ { &gObjOsAnim_0FE4, 1.0f, 0.0f, -1.0f, 0x00, -8.0f },
+    /* 26 */ { &gObjOsAnim_0BFC, 1.0f, 0.0f, -1.0f, 0x00, -8.0f },
 };
 
 typedef struct {
-    /* 0x00 */ u8 unk_0;
-    /* 0x01 */ u8 unk_1;
-    /* 0x02 */ Color_RGBA8 unk_2;
-    /* 0x06 */ u8 unk_6;
-    /* 0x07 */ Color_RGBA8 unk_7;
-    /* 0x0B */ u8 unk_B;
-} EnHyUnknownStruct3; // size = 0xC
-
-static EnHyUnknownStruct3 D_80A722D8[] = {
-    { 0, 0, { 255, 255, 255, 255 }, 0, { 255, 255, 255, 255 }, 0 },
-    { 15, 7, { 255, 255, 255, 255 }, 7, { 255, 255, 255, 255 }, 22 },
-    { 7, 3, { 255, 255, 255, 255 }, 3, { 255, 255, 255, 255 }, 1 },
-    { 3, 2, { 255, 255, 255, 0 }, 2, { 55, 55, 255, 0 }, 15 },
-    { 8, 3, { 0, 0, 0, 0 }, 3, { 255, 0, 0, 0 }, 11 },
-    { 4, 2, { 50, 80, 0, 0 }, 2, { 50, 80, 0, 0 }, 16 },
-    { 10, 4, { 255, 255, 255, 255 }, 4, { 255, 255, 255, 255 }, 10 },
-    { 13, 6, { 0, 50, 160, 0 }, 6, { 255, 255, 255, 0 }, 4 },
-    { 11, 5, { 160, 180, 255, 0 }, 5, { 160, 180, 255, 0 }, 9 },
-    { 2, 2, { 220, 0, 80, 0 }, 2, { 255, 255, 255, 0 }, 13 },
-    { 2, 2, { 0, 130, 220, 0 }, 2, { 255, 255, 255, 0 }, 14 },
-    { 12, 5, { 70, 160, 230, 0 }, 5, { 255, 255, 100, 0 }, 20 },
-    { 5, 2, { 150, 60, 90, 0 }, 2, { 255, 240, 150, 0 }, 18 },
-    { 9, 3, { 200, 180, 255, 0 }, 3, { 200, 180, 255, 0 }, 12 },
-    { 6, 2, { 140, 255, 110, 0 }, 2, { 255, 255, 255, 0 }, 19 },
-    { 14, 6, { 130, 70, 20, 0 }, 6, { 130, 180, 255, 0 }, 21 },
-    { 2, 2, { 255, 255, 255, 255 }, 2, { 255, 255, 255, 255 }, 5 },
-    { 8, 3, { 90, 100, 20, 255 }, 3, { 100, 140, 50, 255 }, 11 },
-    { 1, 1, { 255, 255, 255, 255 }, 1, { 255, 255, 255, 255 }, 6 },
-    { 14, 6, { 160, 0, 100, 0 }, 6, { 70, 130, 210, 0 }, 21 },
-    { 9, 3, { 160, 230, 0, 0 }, 3, { 0, 150, 110, 0 }, 12 },
+    /* 0x00 */ u8 headInfoIndex;       // EnHyHeadIndex
+    /* 0x01 */ u8 unusedSkelInfoIndex; // EnHySkeletonIndex, see EnHy#objBankIndexSkelUnused
+    /* 0x02 */ Color_RGBA8 envColorSeg8;
+    /* 0x06 */ u8 skelInfoIndex; // EnHySkeletonIndex
+    /* 0x07 */ Color_RGBA8 envColorSeg9;
+    /* 0x0B */ u8 animInfoIndex; // EnHyAnimationIndex
+} EnHyModelInfo;                 // size = 0xC
+
+// indexed by EnHyType
+static EnHyModelInfo sModelInfos[] = {
```

Comment:
info as plural is a bit strange i think. goes for the others as well

Reply:
would it be fine to just use `sModelInfo` even though there are several "info structs" inside the array? idk what to call the arrays